internal/cmd/configssh.go

```diff
@@ -212,7 +259,7 @@ func makeSSHConfig(binPath, host, userName, workspaceName, privateKeyFilepath st
 		host := fmt.Sprintf(
 			`Host coder.%s
    HostName coder.%s
-   ProxyCommand %s tunnel %s 12213 stdio
+   ProxyCommand "%s" tunnel %s 12213 stdio
```

Comment:
Do the quotes work on Linux too? I _thought_ I tried it before...

Reply:
Works on windows, mac and linux from my testing.